### ISSUE TITLE
Support fedora 39

### ISSUE THREAD

**Issue by jolars** [abn/throttled-rpm#27]
Could you please consider updating the copr package for fedora 39?

**Comment by IsThisEvenCode** [comment]
I got it working on Fedora 39 again!

Delete throttled according to the manual provided on the github page. Purge the config, too.
After this build it from source. 

Worked for me!

**Comment by maxdebayser** [comment]
Building from source is working for me on 39 too. But it would be cleaner to have a package.

**Comment by nawramm** [comment]
I have forked the repo and built for 39 (https://copr.fedorainfracloud.org/coprs/nawram/throttled/).

**Comment by scr4bble** [comment]
@abn would you find some time to build this for fedora 39 so people don't have to keep migrating over to other repositories to avoid manual building?

**Comment by abn** [comment]
@scr4bble sure happens that I have some time today :D

**Comment by abn** [comment]
Also if anyone is interested in helping maintain please reach out; since I do not actively use this anymore.

**Comment by scr4bble** [comment]
Wonderful. Thank you :) that was super fast response.
I would help but since I have no clue how this works I don't think it would be beneficial to anybody :D

Just to add some context:
I dropped this service (throttled) long ago as I didn't have the CPU throttling issues anymore. But now, after many years it came back (some days ago, during some update in Fedora 38). I am currently upgrading to Fedora 39 to see if the throttling goes away but if not, I will have to install and enable this service again. That's why I came back here.

Just in case somebody has experienced the same and happens to stumble upon my post here.
Laptop: T480
CPU Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
